mobile/package.json

```diff
@@ -1,11 +1,19 @@
 {
   "name": "words-alive-mobile",
   "dependencies": {
+    "@react-native-community/masked-view": "0.1.10",
+    "@react-navigation/bottom-tabs": "^5.11.7",
+    "@react-navigation/native": "^5.9.2",
     "expo": "^40.0.0",
     "react": "16.13.1",
     "react-dom": "16.13.1",
     "react-native": "^0.63.4",
+    "react-native-gesture-handler": "~1.8.0",
+    "react-native-reanimated": "~1.13.0",
+    "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "~2.15.0",
+    "react-native-svg": "^12.1.0",
+    "react-native-svg-transformer": "^0.14.3",
```

Comment:
Just checking... are all of these needed? At the very least I don't think we need the svg libs now that we're using PNGs, right? (also if you remove any remember to `npm install` again so the package lock gets updated)